examples/usb_host_descriptors_find_boot_keyboard.py

```diff
@@ -0,0 +1,45 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025 Tim Cocks for Adafruit Industries
+#
+# SPDX-License-Identifier: MIT
+import array
+
+import usb
+
+import adafruit_usb_host_descriptors
+
+keyboard_interface_index = None
+keyboard_endpoint_address = None
+keyboard = None
+
+# scan for connected USB devices
+for device in usb.core.find(find_all=True):
+    # check for boot keyboard endpoints on this device
+    keyboard_interface_index, keyboard_endpoint_address = (
+        adafruit_usb_host_descriptors.find_boot_keyboard_endpoint(device)
+    )
+    # if a boot mouse interface index and endpoint address were found
```

Comment:
```suggestion
    # if a boot keyboard interface index and endpoint address were found
```

Reply:
whoops, thank you. fixed with latest commit.